tests/integration/env-tests/test_podman_compose_env.py

```diff
@@ -49,6 +49,7 @@ def test_env(self):
     - https://docs.docker.com/compose/how-tos/environment-variables/envvars/
     - https://github.com/compose-spec/compose-spec/blob/main/04-version-and-name.md
     """
+
```

Comment:
Unnecessary change.

Reply:
This was required to make `ruff` happy. Otherwise the formatting check fails. Changed in https://github.com/containers/podman-compose/compare/198623df38e818d8f1f0baac35380d01841d286c..6fd5b545f163a69dd0c45b88d57e3cbead5620d0

Reply:
Understood. I suppose that new line should have been added to the commit that added the comment and the test. Probably not worth the effort at this point.